### PR TITLE
feat (docket): add modal to buy links

### PR DESCRIPTION
**Pull request by albertisfu**
Fixes: #1352

### CODE REVIEW

**Comment by albertisfu** [comment]
Added install recap modal to more buttons:

- Added in rd_download_button.html if recap document is available or not
- Added in pacer_links_dropdown.html View on Pacer main button and toggle dropdown buttons
- Added in docket_li.html for View on Pacer action in tag template
- Added in view_docket.html for Buy Docket on Pacer button when no docket entries available

File: cl/opinion_page/templates/includes/buy_pacer_modal.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      </div>
+      <div class="modal-body">
+        <p>
+            Launched in 2010, the RECAP extension is a free tool for your browser that helps us collect the content you see on CourtListener. Once installed, anything you buy on PACER gets automatically added to the RECAP Archive and anything another RECAP user has bought is automatically free to you.
+        </p>
+      </div>
+      <div class="modal-footer">
+        <p>
+          <a href="https://free.law/recap/" target="_blank"
+             class="btn btn-default">Install RECAP</a>
```
Comment (mlissner) [inline cl/opinion_page/templates/includes/buy_pacer_modal.html:23]:
```suggestion
             class="btn btn-default">Learn More</a>
```

**Review by mlissner — APPROVED** [review]
This looks good to me. Merging after making one more tweak to the modal itself.

**Comment by mlissner** [comment]
The tests are having a minor fit because they don't have the S3 rights they need, but this will soon be fixed by @satsuki-chan, I think.

Thank you Alberto for your first contribution!

**Pull request merged**